src/scripting/lua_map.cc

```diff
@@ -2058,7 +2072,7 @@ int LuaTribeDescription::get_workers(lua_State* L) {
 /* RST
    .. method:: has_building(buildingname)
 
-      Returns true if buildingname is a building and the tribe can use it.
+      Returns true if **buildingname** is a building and the tribe can use it.
```

Comment:
Type markup for true/false. Several more below.

Reply:
fixed